keep.tex

```diff
@@ -229,10 +229,10 @@ \subsection{Keep operations}
 \subsubsection{Creation and population}
 
 A contract, $Contract_{owner}$, requests a keep by publishing a
-request to the blockchain. Once a keep, $Keep_{accepted}$, accepted a
-request and is initialized off-chain, it will respond with a set of
-public keys the calling contract can use to communicate privately with
-the keep.
+request to the blockchain. Once a keep, $Keep_{accepted}$, has accepted a
+request and finished initializing off-chain, it will respond to the request
+with a set of public keys the calling contract can use to communicate privately
```

Comment:
We've been keeping line length to 70

Reply:
! On it, sorry about that. My wrap is at 80, I kind of noticed but forgot to double-check.